install.md

```diff
@@ -0,0 +1,28 @@
+---
+layout: default
+title: Install
+order: 30
+---
+
+# Install or upgrade
+
+mozregression is a Python (currently 2.7) package installable via pip. You can
+follow the OS-specific instructions to install pip if you don't know how to do it:
+
+- **Windows**:
+  Download and install python 2.7 from
+  [ActiveState](http://www.activestate.com/activepython/downloads).
+  It will include pip and set python in your PATH.
```

Comment:
I assume it is simpler for users, because that includes pip and set python in the PATH. Your point is good though, because mozilla-build does not just install python - it also install everything needed to build mozilla projects - but do we need this now that mozcommitbuilder has been removed ?
Maybe we could keep the mozilla-build step in case we want to build things in a mozregression future, instead of just downloading binaries.

Honestly, I am not a windows user, and i don't really mind one of the two options - still I would prefer a pure python approach personnaly, but maybe just because I know that the activestate python is easy to install and works well. Your choice !